qemu/tests/uefishell.py

```diff
@@ -47,6 +48,9 @@ def setup(self, under_fs0):
         params["cdroms"] = "%s %s" % ("uefishell", params["cdroms"])
         params["cdrom_uefishell"] = self.copy_uefishell()
         params["bootindex_uefishell"] = "0"
+        if params.get("secureboot") == "yes":
+            params["secureboot_pk_kek"] %= self.copy_secureboot_pk_kek()
+            params["extra_params"] %= params["secureboot_pk_kek"]
```

Comment:
why not use secureboot_pk_kek as the flag, so we can remove secureboot, if secureboot_pk_kek is set, assign the efi file path as well as fill in the extra_params command line

Reply:
agree with you, thanks